### DISCUSSION TITLE
Syntax error when adding "deb http://pub.freerdp.com/repositories/deb/focal/ freerdp-nightly main"

### DISCUSSION THREAD

**Discussion by gwillegers** [FreeRDP/FreeRDP#8656] [Q&A] [answered]
Because of a microphone issue when connecting from Linux Mint to a Windows 11 computer, I want to install the latest nightly from the PPA. When I try to add the PPA via the Package Sources app, it says there is a syntax error. What is wrong? I am using Linux Mint 20.3 (una) which is based on Ubuntu 20.04 (focal).

**Comment by akallabeth** [comment]
did you follow the instructions correctly?
The link for the installation methods is https://github.com/FreeRDP/FreeRDP/wiki/PreBuilds

**Reply by gwillegers** [reply]
That is the exact route I followed. I selected the link for the nightly for Ubuntu. There it says to add
deb http://pub.freerdp.com/repositories/deb/DIST/ freerdp-nightly main 
replacing DIST with the distro name (in my case focal).
I now pasted it directly in the sources file, instead of using the package sources app. Now the repo is added. I have no idea why the app says there is a syntax error.
xfreerdp --version results in **This is FreeRDP version 2.0.0-dev (git 42d2b69)**
Is this the latest version? The installed package has version 2.6.1+git2022030714.
(I am having underrun errors like described in https://github.com/FreeRDP/FreeRDP/issues/8592.)

**Comment by akallabeth** [comment, marked as answer]
@gwillegers ok, so it works now or what do you mean with syntax error exacty?

also your output should look similar to this (got debian bullseye here):
```
 /opt/freerdp-nightly/bin/xfreerdp --version
This is FreeRDP version 3.0.0-dev (dd0d130f4)
```

**Reply by gwillegers** [reply]
Adding the line
deb http://pub.freerdp.com/repositories/deb/focal/ freerdp-nightly main
directly in the sources file (not via the package sources app)it worked fine. The package source is added and enabled.

I just noticed the package name is different. So I installed freerdp-nightly-dbg. When I run xfreerdp from /opt/freerdp-nightly/bin, is it showing version 3.0.0.

I will test the microphone on the next Teams meeting (which will be tomorrow).
Thanks for your help.
